projects/ngx-datatable/src/lib/utils/throttle.ts

```diff
@@ -1,43 +1,41 @@
 /**
  * Throttle a function
  */
-export function throttle(func: any, wait: number, options?: any) {
+export const throttle = (func: any, wait: number, options?: any) => {
```

Comment:
Why is this changed here? Seems unrelated to the rules?

Reply:
refactored as part of `@typescript-eslint/no-this-alias`